python/chronos/test/bigdl/chronos/forecaster/tf/test_lstm_keras_forecaster.py

```diff
@@ -95,6 +95,31 @@ def test_lstm_forecaster_save_load(self):
         load_model_yhat = self.forecaster.predict(test_data[0])
         assert yhat.shape == (400, 1, 2)
         np.testing.assert_almost_equal(yhat, load_model_yhat, decimal=5)
+    
+    def test_lstm_customized_loss_metric(self):
+        train_data, test_data = create_data(tf_data=True)
+        loss = tf.losses.mean_squared_error
+        def customized_metric(y_true, y_pred):
+            return tf.losses.mean_squared_error(tf.convert_to_tensor(y_pred),
+                                      tf.convert_to_tensor(y_true)).numpy()
+        from bigdl.chronos.forecaster.tf.lstm_forecaster import LSTMForecaster
+        self.forecaster = LSTMForecaster(past_seq_len=10,
+                                    input_feature_num=10,
+                                    output_feature_num=2,
+                                    loss=loss,
+                                    metrics=[customized_metric],
+                                    lr=0.01)
+        self.forecaster.fit(train_data, epochs=2, batch_size=32)
```

Comment:
oh so this means that we can directly support customized loss and metric?

Reply:
> 

yes

Reply:
> oh so this means that we can directly support customized loss and metric?

yes

Reply:
that's great.